src/WebJobs.Script.WebHost/Security/Authentication/AuthenticationLevelOptions.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Authentication;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Authentication
```

Comment:
nit: can you clean up the namespaces in these Security/XXX directories? E.g. Microsoft.Azure.WebJobs.Script.WebHost.Security.Authentication etc.